tests/e2e/Skip_Initial_Tests/composer.json

```diff
@@ -5,13 +5,10 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.5",
-        "phpspec/phpspec": "^4.3",
-        "leanphp/phpspec-code-coverage": "^4.0"
+        "phpunit/phpunit": "^6.5"
```

Comment:
phpspec, codeception must not be tested in the core, as it will result in cycle dependencies in tests between adapters and core

since we don't test phpspec in the core - we don't need it